### PR TITLE
Visualize the commits for all branches

### PR DESCRIPTION
**Pull request by Yuuki77**
Fixes https://github.com/jesseduffield/lazygit/issues/1053

Now I added the command called `allBranchesLogCmd` and you can see the all commit graph
I was not sure where to show the graph so for now I just show it in `Status` panel .
If you have any suggestions, please let me know. I am happy to update it.  

image

### CODE REVIEW

File: pkg/i18n/english.go [hunk 1]
```diff
@@ -666,6 +667,7 @@ func englishTranslationSet() TranslationSet {
 		LcMergeIntoCurrentBranch:            `merge into currently checked out branch`,
 		ConfirmQuit:                         `Are you sure you want to quit?`,
 		SwitchRepo:                          `switch to a recent repo`,
+		AllBranchesLogGraph:                 `show all branch logs`,
```
Comment (jesseduffield) [inline pkg/i18n/english.go:670]: By convention if we have a string start with a lowecase letter we prefix the key with `Lc` to disambiguate from a similar string with an up-cased first letter

**Review by jesseduffield — CHANGES_REQUESTED** [review]
Just the one comment, otherwise looks good! I think at some point we should add tabs to the main panel which appear conditional on which side-bar is selected, but there are some edge cases there that would need to be thought through, so I'm happy with this for now :)

**Comment by Yuuki77** [comment]
@jesseduffield Thank you for the reviews! 😄 
It would be nice to add tabs to the main panel.
As you said, this is good enough for now!
I updated the pr based on your feedback.
Please take a look when you have time 👍

File: pkg/i18n/english.go (hunk 1 above)
Comment (Yuuki77) [reply pkg/i18n/english.go]: updated it https://github.com/jesseduffield/lazygit/pull/1104/commits/8b9f2527947d39a6c52f779a6b3b98e6971cdb02

**Comment by jesseduffield** [comment]
Thanks again!

**Comment by 9mm** [comment]
How do I show this `allBranchesLogCmd` instead of the current "welcome" message inside status?

**Comment by 9mm** [comment]
sorry, nevermind. I see its the `a` key within status. 

so many options, I love it. taking me a min to learn my way around but this is really fast now that ive started remembering everything

**Pull request merged**